clients/csharp-uwp/UWPVoiceAssistantSample/DirectLineSpeechDialogBackend.cs

```diff
@@ -323,13 +339,45 @@ private DialogServiceConfig CreateConfiguration()
                 config.SetProperty(PropertyId.Conversation_Custom_Voice_Deployment_Ids, customVoiceIds);
             }
 
-            if (LocalSettingsHelper.EnableSdkLogging)
+            if (enableSdkLogging)
             {
                 var logPath = $"{ApplicationData.Current.LocalFolder.Path}\\sdklog.txt";
                 config.SetProperty(PropertyId.Speech_LogFilename, logPath);
             }
 
             return config;
         }
+
+        private bool RefreshConfigValues()

```

Comment:
Method name should reflect the return type; `RefreshX()` implies a void. `TryRefreshConfigValues` would make sense.

Reply:
Good call